src/plugins/intel_gpu/src/graph/include/program_node.h

```diff
@@ -301,6 +301,10 @@ struct program_node {
     void set_in_shape_of_subgraph(bool val = true) { in_shape_of_subgraph = val; }
     bool is_in_shape_of_subgraph() const { return in_shape_of_subgraph; }
 
+    void set_state_init_subgraph_id(std::string id) { state_init_subgraph_id = id; }
```

Comment:
I think the function name is a little bit hard to understand. (does it set "init subgraph id" or "set state"..?) 
How about "set_variable_id_of_init_subgraph" ?

Reply:
For set/get funcs naming convention, updated it as `set_state_variable_id_of_init_subgraph`